### PR TITLE
Fix version

### PR DESCRIPTION
**Pull request by iusyk**
The AUTOREVs have been replaced with the actual revisions

Signed-off-by: Ihor Usyk <ihor_usyk@epam.com>

### CODE REVIEW

**Comment by rshym** [comment]
Please fix misprint in commit message: AUOTOREVs ==> AUTOREVs.

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/xt-log/xt-log_git.bbappend [hunk 1]
```diff
@@ -1,2 +1,3 @@
 SRC_URI = "git://github.com/xen-troops/${PN}.git"
-SRCREV = "${AUTOREV}"
+SRCREV = "152eaf422b65eecc5bee63521e4606be21d2cb27"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=1ebbd3e34237af26da5dc08a4e440464"
```
Comment (rshym) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/xt-log/xt-log_git.bbappend:3]: As I understand, we do not need LIC_FILES_CHKSUM to be specified here.
I have checked that we have meta-xt-images fixed in meta-xt-products to version with this checksum (for GPL3).
And xt-log is fixed right here to same GPL3 version.
So, there is no need for LIC_FILES_CHKSUM.

Comment (iusyk) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/xt-log/xt-log_git.bbappend:3]: I agree, but build required it, it was not my idea :(
anyway i will recheck

**Comment by iusyk** [comment]
> Please fix misprint in commit message: AUOTOREVs ==> AUTOREVs.

fixed

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/xt-log/xt-log_git.bbappend (hunk 1 above)
Comment (rshym) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/xt-log/xt-log_git.bbappend:3]: I have checked. This LIC_FILES_CHKSUM is defined in revision
1a19ac5 Add C++ header only log
of file https://github.com/xen-troops/meta-xt-images/commits/master/recipes-domx/meta-xt-images-domx/recipes-extended/xt-log/xt-log_git.bb
We have xt-images 'fixed' to that resision in product's manifest, so no need to redefine it here.

I have built tu2019 with this pull request with removed line "LIC_FILES_CHKSUM ...".
DomD built succesfully.

So, could you please remove line with checksum and merge this PR.

**Comment by rshym** [comment]
LGTM.
Tested with removed checksum for xt-log - works as expected.

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/xt-log/xt-log_git.bbappend (hunk 1 above)
Comment (iusyk) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/xt-log/xt-log_git.bbappend:3]: ok

**Pull request merged**